openlibrary/plugins/upstream/account.py

```diff
@@ -860,6 +860,10 @@ def GET(self):
 class fetch_goodreads(delegate.page):
     path = "/account/import/goodreads"
 
+    @require_login
+    def GET(self):
+        return render['account/import']()
```

Comment:
This is the way redirects work in web.py:

```suggestion
    def GET(self):
        raise web.seeother("/account/import")
```